### PR TITLE
fix docs typos

### PR DESCRIPTION
**Pull request by morleytatro**
## Changes

Just fixing some docs typos.

## How to Review

NA

## Checklist

- [x] Unit tests updated
- [x] `docs/` updated (if necessary)
- [x] `pnpm run update:examples` run (only applicable for openapi-typescript)

### CODE REVIEW

File: packages/openapi-react-query/README.md [hunk 1]
```diff
@@ -25,42 +25,42 @@ Next, generate TypeScript types from your OpenAPI schema using openapi-typescrip
...
 
 ## Usage
 
-Once your types has been generated from your schema, you can create a [fetch client](../openapi-fetch), a react-query client and start querying your API.
+Once your types have been generated from your schema, you can create a [fetch client](../openapi-fetch), a react-query client and start querying your API.
 
 ```tsx
-import createFetchClient from "openapi-fetch";
-import createClient from "openapi-react-query";
-import type { paths } from "./my-openapi-3-schema"; // generated by openapi-typescript
```
Comment (kerwanp) [inline packages/openapi-react-query/README.md:37]: We use double quotes in this repository and I think we should keep documentation with the same rules (default biome ones)

**Review by kerwanp — CHANGES_REQUESTED** [review]
Thanks for the PR! A clean documentation is way more professional. Happy to merge this once the commit is solved

File: packages/openapi-react-query/README.md (hunk 1 above)
Comment (morleytatro) [reply packages/openapi-react-query/README.md]: Updated, thanks. My autoformat on save was the culprit, lol.

File: packages/openapi-react-query/README.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ Next, generate TypeScript types from your OpenAPI schema using openapi-typescrip
 npx openapi-typescript ./path/to/api/v1.yaml -o ./src/lib/api/v1.d.ts
 ```
 
-> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will err on invalid schemas.
+> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will error on invalid schemas.
```
Comment (drwpow) [inline packages/openapi-react-query/README.md:28]: “err“ is correct; that is a verb. “Error” is a noun, so this isn’t correct.

However, this sentence only applies to 6.x, not 7.x (since that will validate). So we could just remove this altogether.

File: packages/openapi-react-query/README.md [hunk 3]
```diff
@@ -42,25 +42,25 @@ const fetchClient = createFetchClient<paths>({
 const $api = createClient(fetchClient);
 
 const MyComponent = () => {
-  const { data, error, isLoading } = $api.useQuery(
+  const { data, error, isPending } = $api.useQuery(
```
Comment (drwpow) [inline packages/openapi-react-query/README.md:45]:
```suggestion
  const { data, error, isLoading } = $api.useQuery(
```

? I’m confused; this should be `isLoading`, shouldn’t it? Why the change?

Comment (morleytatro) [reply packages/openapi-react-query/README.md:45]: https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading

Comment (morleytatro) [reply packages/openapi-react-query/README.md:45]: You have v5 as a peer dependency.

Comment (drwpow) [reply packages/openapi-react-query/README.md:43]: Ah thanks for explaining! I missed that. I think we have some other docs out-of-date, then. This is good for now!

**Review by drwpow — APPROVED** [review]
Great fixes, thank you!
